apps/quick-learn-frontend-e2e/src/cypress/test/Profile.js

```diff
@@ -0,0 +1,52 @@
+class profile {
+  visitTeamPage() {
+    return cy.get('[href="/dashboard/teams"]');
+  }
+
+  userMenu() {
```

Comment:
What is the objective of wrapper a cypress function over here? I don't see any need for this over here.